package.json

```diff
@@ -171,6 +171,7 @@
     "iti": "0.6.0",
     "jest": "29.7.0",
     "jest-environment-jsdom": "29.7.0",
+    "jquery": "3.7.1",
```

Comment:
I added `jquery` to the list of dev dependencies because we _technically_ require it for our WP integration :)